internal/tools/semconvkit/templates/httpconv/http.go.tmpl

```diff
@@ -54,7 +54,7 @@ var (
 		HTTPStatusCodeKey:            semconv.HTTPStatusCodeKey,
 		HTTPTargetKey:                semconv.HTTPTargetKey,
 		HTTPURLKey:                   semconv.HTTPURLKey,
-		HTTPUserAgentKey:             semconv.HTTPUserAgentKey,
+        UserAgentOriginalKey:         semconv.UserAgentOriginalKey,
```

Comment:
Can you please format this file? You should use "tabs" (instead of "spaces") before `UserAgentOriginalKey`  😉

Reply:
Thanks, done!